augur/filters/__init__.py

```diff
@@ -0,0 +1,8 @@
+import pkgutil
+
+filter_modules = []
+
+for module_info in pkgutil.iter_modules(__path__):
+    module = module_info.module_finder.find_loader(module_info.name)[0].load_module()
+    if hasattr(module, "init"):
+        filter_modules.append(module)
```

Comment:
I think a lot of the cleverness elsewhere (module level inits and "is_enabled") can be avoided by just making this an explicit list.